IO-phone/src/app/poll/poll.component.ts

```diff
@@ -1,20 +1,110 @@
 import { Component, OnInit, inject } from '@angular/core';
 import { PollService } from './poll.service';
+import { Answer, Poll, Question, Student } from './model';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from "@angular/material/button";
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { FormGroup, FormBuilder, Validators, ReactiveFormsModule } from '@angular/forms';
 
 @Component({
   selector: 'app-poll',
   standalone: true,
-  imports: [],
+  imports: [
+    CommonModule, ReactiveFormsModule,
+    MatButtonModule, MatFormFieldModule, MatInputModule, MatInputModule
+  ],
   templateUrl: './poll.component.html',
   styleUrl: './poll.component.css'
 })
-export class PollComponent implements OnInit {
+export class PollComponent {
   private pollService = inject(PollService);
+  private formBuilder = inject(FormBuilder);
+
+  pollTemplate: Poll | null = null;
+  questions: Question[] = [];
+
+  students: Student[] = [];
+  currentStudentIndex = -1;
```

Comment:
Tutaj normalnie też bym proponował, żeby inicjalizować jakimś `undefined`, ale widzę, że wymagałoby dodatkowego refactoru, więc nie ma co tracić na to czasu